src/core/Akka.Remote/Endpoint.cs

```diff
@@ -1953,21 +1953,9 @@ private void Reading()
                                     ackAndMessage.MessageOption.SerializedMessage,
                                     ackAndMessage.MessageOption.SenderOptional);
                             }
-                            catch (SerializationException e)
-                            {
-                                LogTransientSerializationError(ackAndMessage.MessageOption, e);
-                            }
-                            catch (ArgumentException e)
-                            {
-                                LogTransientSerializationError(ackAndMessage.MessageOption, e);
-                            }
-                            catch (InvalidCastException e)
-                            {
-                                LogTransientSerializationError(ackAndMessage.MessageOption, e);
-                            }
                             catch (Exception e)
                             {
-                                throw;
+                                LogTransientSerializationError(ackAndMessage.MessageOption, e); 
```

Comment:
Have to just make this a catch all because libraries like Newtonsoft.Json don't throw either of the handled cases above. If you have a serialization problem, it's your fault - we'll log it and not let it blow up the connection. On the user to fix it.